kernel/aster-nix/src/vm/vmo/dyn_cap.rs

```diff
@@ -33,22 +34,25 @@ impl Vmo<Rights> {
     /// so are the access rights.
     pub fn new_slice_child(
         &self,
-        range: Range<usize>,
+        from: Range<usize>,
+        to: Range<usize>,
```

Comment:
I can understand your point, but naming these two ranges as `from` and `to` could easily confuse readers. The meanings of these intervals should respectively represent the range to be inherited within the parent and the range size of the child VMO relative to the parent VMO. The names `from` and `to` might give the initial impression that the range `to` is entirely derived from the `from` interval.